packages/cloudflare/src/workflows.ts

```diff
@@ -24,14 +24,19 @@ import { init } from './sdk';
 
 const UUID_REGEX = /^[0-9a-f]{8}-?[0-9a-f]{4}-?[0-9a-f]{4}-?[0-9a-f]{4}-?[0-9a-f]{12}$/i;
 
-function propagationContextFromInstanceId(instanceId: string): PropagationContext {
-  // Validate and normalize traceId - should be a valid UUID with or without hyphens
-  if (!UUID_REGEX.test(instanceId)) {
-    throw new Error("Invalid 'instanceId' for workflow: Sentry requires random UUIDs for instanceId.");
-  }
+async function hashStringToUuid(input: string): Promise<string> {
```

Comment:
can we add some unit tests for this? Let's just some different kind of inputs.

Reply:
Also changed this to be called `deterministicTraceIdFromInstanceId` since that's what it actually does.